tests/unit/test_analyzer.py

```diff
@@ -61,6 +61,50 @@ def test_inferred_module_type():
     """) == {'boto3': Boto3ModuleType()}
 
 
+def test_recursive_function_none():
```

Comment:
what does the test name ending in `_none` mean?

Reply:
That there are no botocore client calls.